pkg/node/bootstrap.go

```diff
@@ -214,7 +219,12 @@ func bootstrapNode(
 		eventsJSON        []byte
 	)
 
+	logger = passedInLogger
```

Comment:
i find the logger swapping confusing... why not toggle which logger to use by either passing the discarding logger or just `logger` in the constructor like it was before?